iree/hal/vmla/vmla_module.cc

```diff
@@ -858,6 +869,11 @@ static const vm::NativeFunction<VMLAModuleState> kVMLAModuleFunctions[] = {
     vm::MakeNativeFunction("shr.i16", &VMLAModuleState::ShrI16),
     vm::MakeNativeFunction("shr.i32", &VMLAModuleState::ShrI32),
 
+    vm::MakeNativeFunction("iota.i8", &VMLAModuleState::IotaI8),
```

Comment:
so close :P

Reply:
..... I swear this was vscode acting weird.